templates/navigation-bar-new.twig

```diff
@@ -40,18 +40,29 @@
 						data-ga-label="{{ page_category }}">
 							{{ item.title }}
 					</a>
+
+					{% if item.children is not empty %}
+						{% include 'navigation-submenu.twig' with { menu: item } %}
+					{% endif %}
 				</li>
 				{% endfor %}
-				<li class="nav-item nav-donate">
-					<a
-						class="btn btn-donate"
-						href="{{ donatelink }}"
-						data-ga-category="Menu Navigation"
-						data-ga-action="Donate"
-						data-ga-label="{{ page_category }}">
-							{{ donatetext }}
-					</a>
-				</li>
+
+				{% if donate_menu_items|first is not null %}
```

Comment:
Can this use `donate_menu_items is not empty`, [like `item.children`](https://github.com/greenpeace/planet4-master-theme/pull/1628/files?w=0#diff-3ad24a02d9e63bde53c8195a030484bfb200dac551677c03584386ba3394c92aR44)?